.circleci/verbatim-sources/job-specs/binary-job-specs.yml

```diff
@@ -326,6 +326,11 @@
 
   binary_windows_upload:
     <<: *binary_windows_params
+    parameters:
+      executor:
+        type: string
+        default: "windows-medium-cpu-with-nvidia-cuda"
+    executor: <<parameters.executor>>
```

Comment:
Seems like this needs build_environment:
```
build_environment:
  type: string
  default: ""
```

Reply:
Not sure why, but so be it.